pulumi/grapl/README.md

```diff
@@ -44,6 +44,16 @@ according to your anticipated workload.
 pulumi config set postgres-instance-type db.t4g.small
 ```
 
+### `postgres-version`
+
+Not required for local-grapl.
+
+Which version of Postgres to use. Must be >= 13.4.
```

Comment:
Is this version constraint asserted anywhere in the code?

Reply:
It's not. Now that the hard stuff is done I can add a quick major-minor-comparator or something